mage_ai/cluster_manager/kubernetes/workload_manager.py

```diff
@@ -478,30 +543,44 @@ def restart_workload(self, name: str) -> None:
             name, namespace=self.namespace, body={'spec': {'replicas': 1}},
         )
 
-    def configure_pre_start(
+    def create_hooks_config_map(
         self,
         name: str,
-        pre_start_script_path: str,
-        mage_container_config: Dict,
-    ) -> None:
-        self.__validate_pre_start_script(pre_start_script_path, mage_container_config)
+        pre_start_script_path: str = None,
+        mage_container_config: Dict = None,
+        post_start_config: PostStart = None,
+    ) -> Dict:
+        config_map_data = {}
```

Comment:
You want to return or raise Exception if the `mage_container_config` is None, right?